cg/meta/archive/archive.py

```diff
@@ -1,15 +1,34 @@
 import logging
-from collections import namedtuple
 from typing import List, Optional
 
-from cg.apps.cgstats.db.models import Sample
+from housekeeper.store.models import File
```

Comment:
My editor cannot resolve this import, is it possible to fix? Might be an issue with my setup

Reply:
The File import from housekeeper? @seallard

Reply:
Nice, it is being resolved now!